test/studies/shootout/submitted/archive/meteor2.good

```diff
@@ -1,4 +1,7 @@
+meteor2.chpl:302: In function 'goodPiece':
+meteor2.chpl:328: warning: popcount is deprecated - please use popCount instead
 2098 solutions found
+meteor2.chpl:170: called as goodPiece(mask: int(64), pos: int(64))
```

Comment:
This looks wrong to me - the "solutions found" output should happen at execution time, but line 4 looks like a compilation time message.  Not sure why they would interleave.

Did you modify this test because of a failure you encountered or because of a search?  I suspect we don't run it any more (an unmodified run on my machine resulted in: `meteor2.chpl:163: error: halt reached - Cannot bitshift 1 by -5 because -5 is less than 0`)

Reply:
Modified due to a search, I got the same halt when I tried to run it. git history indicates that it is not run anymore either,  I can just undo all changes to it and leave it alone?

Reply:
Sounds good to me :)